src/DrCabal/Cli.hs

```diff
@@ -73,4 +76,10 @@ profileP = do
         , Opt.help "Read profile input from a JSON file, created by 'dr-cabal watch'"
         ]
 
+    profileStyleArgsInput <-  Opt.flag Stacked Stacked $ mconcat
```

Comment:
Let's use the style of `<|>`:

```haskell
profileArgsStyle <- stackedP <|> pure Stacked
```

So later it'll become:

```haskell
profileArgsStyle <- timelineP <|> stackedP <|> pure Stacked
```

Where `stackedP` and `timelineP` are defined using `Opt.flag'`

Reply:
Nice. I didn't find an example to make the default value. Thanks.

Reply:
Done.